Python/Product/PythonTools/PythonTools/Refactoring/LocationPreviewItem.cs

```diff
@@ -103,7 +111,7 @@ class LocationPreviewItem : IPreviewItem {
             }
 
             start = loc.StartColumn - 1;
-            length = origName.Length;
+            length = (loc.EndLine == loc.StartLine ? loc.EndColumn - loc.StartColumn : null) ?? origName.Length;
```

Comment:
`length = loc.EndLine == loc.StartLine ? loc.EndColumn - loc.StartColumn : origName.Length`?

Reply:
`loc.EndLine` and `loc.EndColumn` are `int?` typed, so there's no avoiding the `??` here.